Gemfile

```
@@ -10,15 +10,14 @@ gem 'rails-controller-testing'
 spree_opts = if ENV['SPREE_PATH']
                 { 'path': ENV['SPREE_PATH'] }
              else
-                { 'github': 'spree/spree', 'branch': '5-3-stable' }
+                { 'github': 'spree/spree', 'branch': 'main', 'glob': 'spree/**/*.gemspec' }
```

RuboCop (1.85.1)

[convention] Use 2 (not 3) spaces for indentation.

(Layout/IndentationWidth)